data/json/vehicles/helicopters.json

```diff
@@ -398,7 +398,7 @@
   {
     "id": "SmallFlier2_Wreck",
     "type": "vehicle",
-    "name": "SmallFlier2 Wreck",
+    "name": "Small helicopter",
```

Comment:
This is the only helicopter that doesn't currently have "helicopter" in either the id or name. Also it doesn't follow the naming convention of the other two SmallFlier wrecks.